incubator/hnc/internal/kubectl/describe.go

```diff
@@ -67,6 +73,34 @@ var describeCmd = &cobra.Command{
 			fmt.Printf("  No children\n")
 		}
 
+		// Events
+		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+		defer cancel()
+		events, _ := k8sClient.CoreV1().Events(nnm).List(ctx, metav1.ListOptions{})
+		if events.Size() == 0 {
+			fmt.Printf("  No recent events\n")
```

Comment:
Maybe "No recent HNC events for objects in this namespace" to be even more clear?

Reply:
done